beacon/weakcoin/weak_coin_test.go

```diff
@@ -337,6 +364,12 @@ func TestWeakCoinNextRoundBufferOverflow(t *testing.T) {
 	require.True(t, wc.Get(context.Background(), epoch, nextRound))
 }
 
+type vrfVerifierAdapter struct{}
+
+func (m vrfVerifierAdapter) Verify(nodeID types.NodeID, msg, sig []byte) bool {
```

Comment:
why is this needed?

Reply:
When I wrote that code there VRFVerifier was just a function so I needed to wrap it into an object to satisfy the interface. Since then I created a `VRFVerifier` type in the `signing` package and this is not needed any more.

Removed.